internal/auth0/client/global_data_source.go

```diff
@@ -11,14 +11,17 @@ import (
 
 // NewGlobalDataSource will return a new auth0_global_client data source.
 func NewGlobalDataSource() *schema.Resource {
-	return &schema.Resource{
+	resource := &schema.Resource{
 		ReadContext: readDataGlobalClient,
 		Schema:      globalDataSourceSchema(),
 		Description: "Retrieve a tenant's global Auth0 application client.",
-		DeprecationMessage: "This resource has been deprecated in favour of the newly introduced `auth0_pages` " +
-			"resource and it will be removed in a future version." +
+		DeprecationMessage: "This resource has been deprecated in favour of the `auth0_pages` resource and it will be removed in a future version." +
```

Comment:
```suggestion
		DeprecationMessage: "This resource has been deprecated in favour of the `auth0_pages` resource and it will be removed in a future version. " +
```